deflect/StreamSendWorker.cpp

```diff
@@ -164,6 +164,20 @@ Stream::Future StreamSendWorker::enqueueClose()
     return _enqueueRequest({[this] { return _send(MESSAGE_TYPE_QUIT, {}); }});
 }
 
+Stream::Future StreamSendWorker::enqueueObserverOpen()
+{
+    return _enqueueRequest({[this] {
+        return _send(MESSAGE_TYPE_OBSERVER_OPEN,
+                     QByteArray::number(NETWORK_PROTOCOL_VERSION));
+    }});
+}
+
+Stream::Future StreamSendWorker::enqueueObserverClose()
+{
+    return _enqueueRequest(
+        {[this] { return _send(MESSAGE_TYPE_OBSERVER_QUIT, {}); }});
```

Comment:
do we need a separate MESSAGE_TYPE_OBSERVER_QUIT? Couldn't we just use the normal MESSAGE_TYPE_QUIT?